src/dropdown/dropdown.stories.ts

```diff
@@ -66,7 +66,8 @@ export default {
 		dropUp: false,
 		size: "md",
 		theme: "dark",
-		fluid: false
+		fluid: false,
+		menuIsClosed: true
```

Comment:
I've added the new @Input property menuIsClosed to the documentation. However, during my code review, it appeared to be used as internal state, and I'm unsure whether it should be exposed at the docs. @Akshat55

Reply:
```suggestion
		isOpen: false
```